### ISSUE TITLE
Add support for calling the Portal endpoints

### ISSUE THREAD

**Issue by ikethecoder** [bcgov/gwa-cli#33]
The Portal exposes the following APIs that should be available to call from the command line interface:

- PUT /ds/api/v1/namespaces/{ns}/contents
- PUT /ds/api/v1/namespaces/{ns}/datasets
- PUT /ds/api/v1/namespaces/{ns}/products
- PUT /ds/api/v1/namespaces/{ns}/issuers
 
They each take a JSON payload, but the CLI should accept YAML and convert to JSON.

An example YAML for `/ds/api/namespaces/{ns}/products` is:

```
kind: Product
appId: 2994C28E08PA
name: My API
environments:
  - id: 1F7AA900
    name: dev
    active: false
    approval: false
    flow: kong-api-key-acl
    additionalDetailsToRequest: Please provide a bit more of this
    services: []
```

Commands could be something like:

```
gwa put-content or gwa pc for short
gwa put-dataset or gwa pd for short
gwa put-product or gwa pp for short
gwa put-issuer or gwa pi for short
```

There is also a requirement to be able to add the contents of a file into the JSON for `put-content`.  Perhaps something like:

```
gwa put-content -content ./mydoc.md mydoc_meta.yaml
```

Where `mydoc_meta.yaml` would be:

```
kind: Content
title: Getting Started with Example API
description: Getting Started with Example API
externalLink: https://github.com/bcgov/xyz/getting_started.md
order: 1
tags: [ ns.xyz ]
isComplete: true
isPublic: true
publishDate: "2021-06-02T08:00:00.000-08:00"
```

The contents of `mydoc.md` would be added to the above as an attribute `content`.

**Comment by ikethecoder** [comment]
Already implemented in #37

**Comment by ikethecoder** [comment]
Already implemented in #37
